assignment2/Assignment2.java

```diff
@@ -3,12 +3,12 @@
 import java.util.Set;
 import java.util.TreeSet;
 import assignment2.data.User;
-import assignment2.view.InitPromptBarView;
+import assignment2.view.ApplicationView;
 
 public class Assignment2 {	 
 	 public static void main(String[] args) 
 	 {
 		 Set<User> users = new TreeSet<User>();
-		 InitPromptBarView.initPrompt(users);
+		 ApplicationView.init(users);
```

Comment:
What do you mean by `ApplicationView` here ? The names should be legible enough to a layman.
Any actual reason you have initialised the users outside your application init ?

Reply:
Changed the name to  `UserApplication` and initialised the user inside `UserApplication.init();`